projects/graph/src/graph.py

```diff
@@ -111,4 +114,31 @@ def bfs_path(self, starting_vertex_id, destination_vertex_id):
               visited.add(v)
               # Then, put all of it's children into the queue
               for neighbor in self.vertices[v]:
-                  q.enqueue(neighbor)
\ No newline at end of file
+                  q.enqueue(neighbor)
+
+        if q.size() == 0 and destination_vertex_id not in visited:
+            print("Destination vertex does not exist.")
+            return
+    def dfs_path(self, starting_vertex_id, destination_vertex_id):
+        # Create an empty stack
+        s = Stack()
+        # Create an empty set of visited vertices
+        visited = set()
+        # Put the starting vertex in our Stack
+        s.push(starting_vertex_id)
+        # While the stack is not empty....
+        while s.size() > 0:
+            # Pop the top node from the stack
+            v = s.pop()
+            # If that node has not been visted...
+            if v not in visited:
+                # Mark it as visited
+                print(v)
+                if destination_vertex_id == v: break
```

Comment:
Per the specs, you want to return a path that leads from starting_vertex_id to destination_vertex_id, so you need a way to keep track of paths and then return a path once you reach the destination